styles.css

```diff
@@ -1,30 +1,25 @@
+
+body {
+  margin: 5%;
+}
```

Comment:
-[ ] As requested by the previous reviewer, kindly remove extra space around your sections. Since you are almost out of time, I believe if you follow the following steps should resolve that.
- Set `margin: 0` on the body element
- Set `width: 100%` on the `.work p` class (Preferably always use percentages for setting sizes of sections in future projects)

I coloured the areas that need extra spaces to be removed
![image](https://user-images.githubusercontent.com/29071984/170835563-57947ff4-0592-473f-9144-9473d2762fd8.png)